src/styles/_custom.scss

```diff
@@ -241,6 +241,16 @@ div.is-rounded img {
   filter: invert(1) contrast(1.3);
 }
 
+.jf-link-article {
```

Comment:
(for later) wondering if we should integrate this/canonicize it in the design system in some way?

Reply:
Yeah good idea. I think there are a few other link styles also to override some weird bulma defaults. I'll make a card for that